checkov/cdk/checks/python/WAFEnabled.yaml

```
@@ -0,0 +1,13 @@
+metadata:
+  version: 0.2
+  approach: define failing
+  id: CKV_AWS_68
+  name: CloudFront Distribution should have WAF enabled
+  category: ENCRYPTION
+scope:
+  languages:
+    - python
+definition:
+  pattern: aws_cdk.aws_cloudfront.CfnDistribution(<ANY>)
+  conditions:
+    - not_pattern: 'aws_cdk.aws_cloudfront.CfnDistribution(<ANY>, distribution_config={<ANY>, "webAclId": <ANY> , <ANY>} , <ANY>)'
```

```suggestion
    - not_pattern: 'aws_cdk.aws_cloudfront.CfnDistribution(<ANY>, distribution_config={<ANY>, "webAclId": $ARG , <ANY>} , <ANY>)'
```

---

Also, Pls try it like this:
```suggestion
    - not_pattern: 'aws_cdk.aws_cloudfront.CfnDistribution(<ANY>, distribution_config={"webAclId": $ARG} , <ANY>)'
```